reasoner/controller/RootConjunctionController.java

```diff
@@ -63,13 +65,6 @@ processorDriver, driver(), processorContext(), bounds, plan(bounds.concepts().ke
         );
     }
 
-    @Override
```

Comment:
No longer needed, since these can be answered by the conjunction in ConjunctionController.